src/Util/GetFirstDocComment.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Roave\BetterReflection\Util;
+
+use PhpParser\NodeAbstract;
+
+/**
+ * @internal
+ */
+final class GetFirstDocComment
+{
+
+    public static function forNode(NodeAbstract $node): string
```

Comment:
Last bit: could you add an isolated test for this?

Test should use a mock node:

 * with comment
 * without comment
 * with mixed comment types
 * with multiple Doc comments